### PR TITLE
Hopefully this is right

### PR DESCRIPTION
**Pull request by Amic4353**
Lemme know!

### CODE REVIEW

File: say_hello.js [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
 
+
 // check all these:
-	       
 say_hello("colman");
 say_hello("amaris");
-say_helly("sam");
-say_hello("alex");
+say_hello("sam");
+say_hello("alex")
```
Comment (colmanhumphrey) [inline say_hello.js]: would like my semicolon back :)
```suggestion
say_hello("alex");
```

File: second_max_index.js [hunk 2]
```diff
@@ -11,15 +11,30 @@
...
     // and the second highest will now be the max!
-    let second_max = Math.max(...array_to_index);
+    let second_max = Math.max(...arrMaxFilter);
     let ind_second = array_to_index.indexOf(second_max);
 
-    return array_to_return_value[ind_second];
+    let arrayToIndexLength = array_to_index.length;
+    let arrayToReturnLength = array_to_return_value.length;
+
+    if (arrayToIndexLength > arrayToReturnLength){
```
Comment (colmanhumphrey) [inline second_max_index.js]: I like the idea here! But for now, let's just check that the arrays are the same length. Something like (pseudocode):
```
if(arrays_not_the_same){
     throw new Error("Arrays must be the same length");
}
```

I think these throws are better than the simpler:

```
if(arrays_not_the_same){
     throw "Arrays must be the same length";
}
```

**Review by colmanhumphrey — CHANGES_REQUESTED** [review]
Looks great! Minor changes

**Comment by Amic4353** [comment]
HEY! made the changes you asked for, master.

**Review by colmanhumphrey — APPROVED** [review]
Love the new trycatch blocks on the comments!

I'd probably throw that particular error right at the start, but no harm.

**Pull request merged**